MapboxAndroidDemo/proguard-rules.pro

```diff
@@ -40,3 +40,35 @@
 
 # MAS data models that will be serialized/deserialized over Gson
 -keep class com.mapbox.services.api.directionsmatrix.v1.models.** { *; }
+
+# --- GMS ---
```

Comment:
We should remove `GMS` rules because Proguard rules are properly managed upstream mapbox/mapbox-events-android#109 
We're using Maps SDK `6.0.0-beta.7` version which includes Events `3.0.0` version with the fix 😉